backend/src/controllers/confirmation.controller.ts

```diff
@@ -2,7 +2,7 @@ import { Order } from './../models/order.model';
 import { Request, Response } from 'express';
 import jwt from 'jsonwebtoken';
 
-function parseJwt(token: string): any {
+const parseJwt = (token: string): any => {
```

Comment:
If your function has only a return row. You can type like:

const parseJWT = (token: string) => JSON.parse(Buffer.from(token.split('.')[1], 'base64').toString());